metron-platform/metron-common/src/main/java/org/apache/metron/common/Constants.java

```diff
@@ -43,6 +43,7 @@
     ,DST_PORT("ip_dst_port")
     ,PROTOCOL("protocol")
     ,TIMESTAMP("timestamp")
+    ,ORIGINAL("original_string")
```

Comment:
I agree. I think we've needed something like this for a while. it will help standardize our parsing. great job noticing this.  I am definitely +1 on refactoring grok to look like this